### PR TITLE
Remove duplicate path among composite swaggers

### PR DESCRIPTION
**Pull request by sarangan12**
This PR is to fix the issue with the arm-recoveryservices swagger. This is a composite swagger that points to 5 swagger files. The Operations path with 'get' operation is defined in each one of them that causes the autorest core to error out. You will see the following error:

```
C:\workspace\azure-rest-api-specs>"C:\Program Files\nodejs\node.exe" C:\workspace\autorest\src\core\AutoRest\bin\Debug\netcoreapp1.0\node_modules\autorest-core\app.js -i C:\workspace\azure-rest-api-specs\arm-recoveryservices\compositeRecoveryServicesClient.json -pv 0.10.0 -n Azure::ARM::RecoveryServices -pn azure_mgmt_recovery_services -g Azure.Ruby -o C:\workspace\lib -m CompositeSwagger
FATAL: swagger-document/compose - FAILED
Process() Cancelled due to exception : '$.paths["/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations"].get.parameters' has incomaptible values (---
- $ref: '#/parameters/SubscriptionId'
- name: api-version
  in: query
  description: Client Api Version.
  required: true
  type: string
  enum:
    - '2016-12-01'
- $ref: '#/parameters/ResourceGroupName'
, ---
- $ref: '#/parameters/SubscriptionId'
- name: api-version
  in: query
  description: Client Api Version.
  required: true
  type: string
  enum:
    - '2016-06-01'
- $ref: '#/parameters/ResourceGroupName'
).
Error: '$.paths["/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/operations"].get.parameters' has incomaptible values (---
- $ref: '#/parameters/SubscriptionId'
- name: api-version
  in: query
  description: Client Api Version.
  required: true
  type: string
  enum:
    - '2016-12-01'
- $ref: '#/parameters/ResourceGroupName'
, ---
- $ref: '#/parameters/SubscriptionId'
- name: api-version
  in: query
  description: Client Api Version.
  required: true
  type: string
  enum:
    - '2016-06-01'
- $ref: '#/parameters/ResourceGroupName'
).
    at Merge (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:64:9)
    at Merge (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:58:23)
    at Merge (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:58:23)
    at Merge (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:58:23)
    at Merge (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:58:23)
    at Object.MergeYamls (C:\workspace\autorest\src\autorest-core\lib\source-map\merging.ts:212:20)
    at Object.<anonymous> (C:\workspace\autorest\src\autorest-core\lib\pipeline\swagger-loader.ts:418:24)
    at next (native)
    at fulfilled (C:\workspace\autorest\src\autorest-core\lib\polyfill.min.js:27:62)
    at process._tickCallback (internal/process/next_tick.js:103:7)

```
Now with this PR, the issue is resolved and the SDK is generated successfully.

### CODE REVIEW

**Review by vishrutshah — APPROVED** [review]
Changes LGTM! I'd just wait for @dragonfly91 to approve as well so that just they are aware

**Comment by dragonfly91** [comment]
@DheerendraRathor can you please check this PR?

**Comment by DheerendraRathor** [comment]
@sarangan12 This will certainly unblock composite swagger generation issue. Also I'll advise to keep operations api in vaults.json as it is the primary spec here ( @dragonfly91 @sonathan please verify this statement). 

@sonathan The operations api is not correct here. It needs fix in future. See https://github.com/Azure/azure-rest-api-specs/blob/master/arm-recoveryservicesbackup/2016-08-10/swagger/operations.json

**Pull request merged**
